website/content/en/preview/getting-started/getting-started-with-terraform/_index.md

```diff
@@ -108,42 +108,57 @@ module "vpc" {
   single_nat_gateway     = true
   one_nat_gateway_per_az = false
 
+  public_subnet_tags = {
+    "kubernetes.io/cluster/${local.cluster_name}" = "shared"
+    "kubernetes.io/role/elb"                      = 1
+  }
+
   private_subnet_tags = {
-    "kubernetes.io/cluster/${local.cluster_name}" = "owned"
-    # Tags subnets for Karpenter auto-discovery
-    "karpenter.sh/discovery" = local.cluster_name
```

Comment:
This karpenter unique tag requirement is lifted with https://github.com/terraform-aws-modules/terraform-aws-iam/pull/247 - users just need to specify a `subnetSelector` that will match the subnets they intend to use. VPCs and Subnets are shared resources and we shouldn't impose specific tagging on those